src/main/java/com/solvd/meta/carina/gui/pages/DetailsProductPage.java

```diff
@@ -14,6 +17,12 @@ public class DetailsProductPage {
     private ExtendedWebElement price;
     @FindBy(xpath = "//button[@class='btn btn_primary btn_small btn_inventory']")
     private ExtendedWebElement addToCart;
+    @FindBy(xpath = "//div[@class='primary_header']")
+    private Header header;
+
+    public DetailsProductPage(WebDriver driver) {
+        super(driver);
+    }
 
     public ExtendedWebElement getItemName() {
```

Comment:
Do not return ExtendedWebElement from Page objects. For example in DetailsProductPage.java for in getPrice lets return not ExtendedWebElement price, but String with text from price element. Check it for all methods, fix if needed
<img width="362" alt="image" src="https://user-images.githubusercontent.com/103670717/163967968-3835ae27-442b-472a-b0a2-1447ae6528de.png">